tests/service/data/SyntaxTree/Expression/Record - Anon 06.fs.bsl

```diff
@@ -13,7 +13,7 @@ ImplFile
                          (Atomic, false, Ident f, Const (Unit, (3,4--3,6)),
                           (3,3--3,6)), (3,6--3,7),
                        SynLongIdent ([F], [], [None]), (3,3--3,8)),
-                    Offside ((3,8--3,8), None)), [], (3,0--3,11),
+                    Semicolon ((3,8--3,8), (3,8))), [], (3,0--3,11),
```

Comment:
There is no semicolon in the expression.

Reply:
Fixed